domain-model-assistant/Assets/Components/Scripts/Edge.cs

```diff
@@ -319,6 +327,7 @@ public void SpawnPopupLineMenu()
         }
         else
         {
+            this.popupLineMenu.transform.position = this.mousePos + new Vector3(70, -110, 0);
```

Comment:
Extract 🪄 magic numbers to constants! In a future pull request, add logic to make this offset work at all screen sizes and zoom levels